### PR TITLE
Don't run DCM workflow when merging to master

### PR DESCRIPTION
**Pull request by elliette**
Fixes https://github.com/flutter/devtools/issues/5797

Changes introduced:
- only runs DCM workflow on PRs, not when merging to master
- adds the run-dcm-workflow label for daily bump PRs

### CODE REVIEW

**Comment by kenzieschmoll** [comment]
Does the DCM workflow need to be run for this PR?

**Comment by elliette** [comment]
> Does the DCM workflow need to be run for this PR?

Ah, we were skipping the label reminder if there were no changes to Dart files (which is why that never ran). Now we are also skipping the DCM workflow itself if there are no changes to Dart files.

File: .github/workflows/daily-dev-bump.yaml [hunk 1]
```diff
@@ -107,6 +107,7 @@ jobs:
           # Change github credentials back to the actions bot.
           GH_TOKEN="$ORIGINAL_GH_TOKEN"
 
+          gh pr edit $PR_URL $FLAGS --add-label "run-dcm-workflow"
           gh pr edit $PR_URL $FLAGS --add-label "autosubmit"
```
Comment (kenzieschmoll) [inline .github/workflows/daily-dev-bump.yaml:111]: nit: I think this can be a comma separated list:
`"autosubmit,run-dcm-workflow"`

**Review by kenzieschmoll — APPROVED** [review]
lgtm

**Pull request merged**